en_US.json

```diff
@@ -1256,6 +1256,7 @@
 	"hint.variable.$realtime_log_ds_ratio": "RT log downsample ratio 0.xx",
 	"hint.variable.$remote_user": "user name extracted from Basic auth",
 	"hint.variable.$request": "full HTTP request line",
+	"hint.variable.$request_cpu_time": "CPU time in nanoseconds) spent processing the request before transferring the response body",
```

Comment:
The closing parenthesis should be removed.

Reply:
Thanks. I have updated.